page/Views/layout.php

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="CodeIgniter Dev Team">
+    <title>CodeIgniter Translation Status</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+</head>
+<body>
+
+<main>
+    <div class="container py-4">
+        <header class="pb-3 mb-4 border-bottom">
+            <a href="/translations" class="d-flex align-items-center text-dark text-decoration-none">
```

Comment:
I tested this locally and clicking the list directs me to my directory root.
```suggestion
            <a href="index.html" class="d-flex align-items-center text-dark text-decoration-none">
```